api/v1/peer/peer.pb.go

```diff
@@ -148,6 +148,9 @@ type ChangeNotification struct {
 	Name string `protobuf:"bytes,1,opt,name=name,proto3" json:"name,omitempty"`
 	// Address is the address of the peer's gRPC service.
 	Address string `protobuf:"bytes,2,opt,name=address,proto3" json:"address,omitempty"`
+	// TLS provides information to connect to the Address with TLS enabled.
+	// If not set, TLS shall be assumed to be disabled.
+	Tls *TLS `protobuf:"bytes,4,opt,name=tls,proto3" json:"tls,omitempty"`
```

Comment:
Nit: this should be squashed with the previous commit

Reply:
I've split this into 2 commits so that it's easier to backport to v1.8.